nav2_map_server/include/nav2_map_server/occ_grid_loader.hpp

```diff
@@ -29,6 +28,18 @@
 
 namespace nav2_map_server
 {
+
+typedef struct
```

Comment:
`struct LoadParameters` please

Reply:
Fixed in https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-f278c22b4e54afc4d64216a9c1e5e802R32